addon/components/rdfa-date-plugin/date-time-picker.ts

```diff
@@ -7,25 +7,25 @@ import { localCopy } from 'tracked-toolbox';
 import Intl from 'ember-intl/services/intl';
 
 type Args = {
-  value: Date;
+  value?: Date;
   onChange: (date: Date) => void;
 };
 
 export default class RdfaDatePluginDateTimePicker extends Component<Args> {
   @service declare intl: Intl;
   // eslint-disable-next-line @typescript-eslint/no-unsafe-call
-  @localCopy('args.value') declare date: Date;
+  @localCopy('args.value') declare date?: Date;
 
   get hours() {
-    return this.date.getHours();
+    return this.date?.getHours();
   }
 
   get minutes() {
-    return this.date.getMinutes();
+    return this.date?.getMinutes();
   }
 
   get seconds() {
-    return this.date.getSeconds();
+    return this.date?.getSeconds();
```

Comment:
This makes sense, but why is it needed if a default is given everywhere (so date shouldn't be undefined anymore)? Future proofing?

Reply:
Yes indeed, a default is given everywhere. This is just done for future proofing.